frontend/packages/pipelines-plugin/src/components/pipelineruns/detail-page-tabs/PipelineRunLogs.tsx

```diff
@@ -47,15 +47,20 @@ class PipelineRunLogsWithTranslation extends React.Component<
       const { obj, activeTask } = this.props;
       const taskRunFromYaml = _.get(obj, ['status', 'taskRuns'], {});
       const taskRuns = this.getSortedTaskRun(taskRunFromYaml);
-      const activeItem = this.getActiveTaskRun(taskRuns, activeTask);
+      const activeItem = this.getActiveTaskRun(obj, taskRuns, activeTask);
       this.state.navUntouched && this.setState({ activeItem });
     }
   }
 
-  getActiveTaskRun = (taskRuns: string[], activeTask: string): string =>
-    activeTask
-      ? taskRuns.find((taskRun) => taskRun.includes(activeTask))
+  getActiveTaskRun = (obj: PipelineRunKind, taskRuns: string[], activeTask: string): string => {
+    const activeTaskRun = _.findKey(
+      obj.status.taskRuns,
```

Comment:
Status is an optional field,  there can be lag  sometimes in receiving the status when you start a pipelineRun so it is safe to have an optional chaining here.

```suggestion
      obj?.status?.taskRuns,
```